src/v2/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -125,69 +86,121 @@ export const SavedSearchAlertModal: React.FC<SavedSearchAlertFormProps> = ({
+                </Button>
+              }
             >
-              Save Alert
-            </Button>
-          }
-        >
-          <Join separator={<Spacer mt={4} />}>
-            <Input
-              title="Name"
-              name="name"
-              placeholder={namePlaceholder}
-              value={formik.values.name}
-              onChange={formik.handleChange("name")}
-              onBlur={formik.handleBlur("name")}
-              error={formik.errors.name}
-              maxLength={75}
-            />
-
-            <Box>
-              <Text variant="xs" textTransform="uppercase">
-                Filters
-              </Text>
-              <Spacer mt={2} />
-              <Flex flexWrap="wrap" mx={-0.5}>
-                <Pills items={pills} onDeletePress={handleRemovePillPress} />
-              </Flex>
-            </Box>
-
-            <Box>
-              <Box display="flex" justifyContent="space-between">
-                <Text>Email Alerts</Text>
-                <Checkbox
-                  onSelect={handleToggleNotification("email")}
-                  selected={formik.values.email}
+              <Join separator={<Spacer mt={4} />}>
+                <Input
+                  title="Name"
+                  name="name"
+                  placeholder={namePlaceholder}
+                  value={values.name}
+                  onChange={handleChange("name")}
+                  onBlur={handleBlur("name")}
+                  error={errors.name}
+                  maxLength={75}
                 />
-              </Box>
-              <Spacer mt={4} />
-              <Box display="flex" justifyContent="space-between">
-                <Text>Mobile Alerts</Text>
-                <Checkbox
-                  onSelect={handleToggleNotification("push")}
-                  selected={formik.values.push}
+
+                <Box>
+                  <Text variant="xs" textTransform="uppercase">
+                    Filters
+                  </Text>
+                  <Spacer mt={2} />
+                  <Flex flexWrap="wrap" mx={-0.5}>
+                    <Pills
+                      items={pills}
+                      onDeletePress={handleRemovePillPress}
+                    />
+                  </Flex>
+                </Box>
+
+                <Box>
+                  <Box display="flex" justifyContent="space-between">
+                    <Text>Email Alerts</Text>
+                    <Checkbox
+                      onSelect={selected => setFieldValue("email", selected)}
+                      selected={values.email}
+                    />
+                  </Box>
+                  <Spacer mt={4} />
+                  <Box display="flex" justifyContent="space-between">
+                    <Text>Mobile Alerts</Text>
+                    <Checkbox
+                      onSelect={selected => setFieldValue("push", selected)}
+                      selected={values.push}
+                    />
```

Comment:
Checkbox accepts children which it will use as the label: `<Checkbox {...}>Mobile Alerts</Checkbox>`

Reply:
If we use this approach, we will get a slightly different result that is expected by design

![image](https://user-images.githubusercontent.com/3513494/149803800-bb056e66-3242-46b5-aef3-4d90760b7f0b.png)

Reply:
Ah, I see; in that case make sure you're always using a v3 variant with `Text` in this case; probably `md`

Reply:
Though — I do wonder how we wound up with a layout that's not specified in the design system. :\ — I'd pose that question to design.

Reply:
> Ah, I see; in that case make sure you're always using a v3 variant with `Text` in this case; probably `md`

Done

Reply:
> Though — I do wonder how we wound up with a layout that's not specified in the design system. :\ — I'd pose that question to design.

Asked this question [here](https://www.figma.com/file/zbEFZqjKgKquZwtU8U44j9?node-id=3970:9533#141304787)